modules/dynamic-plugins/con-overriding-core-backend-services.adoc

```diff
@@ -0,0 +1,101 @@
+[id="overriding-core-backend-services_{context}"]
+= Overriding Core Backend Service Configuration
+
+The {product} ({product-very-short}) backend platform consists of a number of core services that are well encapsulated.  The {product-very-short} backend installs these default core services statically during initialization. 
+
+You can configure these core services by customizing the backend source code and rebuilding your {product-short} application. Alternatively, you can customize a core service by installing it as a `BackendFeature` by using dynamic plugin functionality.
+
+To use dynamic plugin functionality to customize a core service in your RHDH application, you must configure the backend to avoid statically installing a given default core service
```

Comment:
"To use dynamic plugin functionality to customize a core service in your RHDH application, you must configure the backend to avoid statically installing a given default core service"
I would suggest using "To use the dynamic plugin functionality..." - to use an article before "dynamic plugin"
And the period is missing for that line.